public/js/components/Frames.js

```diff
@@ -10,7 +10,9 @@ const { endTruncateStr } = require("../util/utils");
 const { basename } = require("../util/path");
 const { getFrames, getSelectedFrame, getSource } = require("../selectors");
 
-require("./Frames.css");
+if (typeof window == "object") {
+  require("./Frames.css");
+}
```

Comment:
I accidentally merged the last PR with touching `require`. I will do this soon. Probably, before this lands.